From: Linus Torvalds <torvalds@macmini.osdl.org>
Date: Tue, 11 Jul 2006 05:21:43 +0000 (-0700)
Subject: x86 MacMini: make built-in speaker sound actually work
X-Git-Tag: v2.6.18-rc2
X-Git-Url: http://kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=7c3dec0679c66ce177726802adbe2f403942fc27

x86 MacMini: make built-in speaker sound actually work

The MacMini board table seems to largely look like any bog-standard
Intel 945 board.

Signed-off-by: Linus Torvalds <torvalds@osdl.org>
---

--- a/sound/pci/hda/patch_sigmatel.c
+++ b/sound/pci/hda/patch_sigmatel.c
@@ -351,6 +351,7 @@ static unsigned int *stac922x_brd_tbl[ST
 	[STAC_REF] =	ref922x_pin_configs,
 	[STAC_D945GTP3] = d945gtp3_pin_configs,
 	[STAC_D945GTP5] = d945gtp5_pin_configs,
+	[STAC_MACMINI] = d945gtp5_pin_configs,
 	[STAC_D965_2112] = d965_2112_pin_configs,
 };
 
